rolling/distribution.yaml

```diff
@@ -7130,6 +7130,22 @@ repositories:
       url: https://github.com/rt-net/rt_manipulators_cpp.git
       version: ros2
     status: maintained
+  rt_usb_9axisimu_driver:
+    doc:
+      type: git
+      url: https://github.com/rt-net/rt_usb_9axisimu_driver.git
+      version: jazzy
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/ros2-gbp/rt_usb_9axisimu_driver-release.git
+      version: 3.0.0-1
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/rt-net/rt_usb_9axisimu_driver.git
+      version: jazzy
```

Comment:
@KuraZuzu just to confirm, you intend to release into ROS 2 Rolling from `jazzy` branch of this repo?

Reply:
Thank you for your confirmation. Sorry for the slight confusion, but there is no error in the specification of the "jazzy" branch, and there is no issue. Could you kindly proceed with the release to ROS 2 Rolling as it is?